src/Pipedrive.net/Models/Common/CustomFields/IntCustomField.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Pipedrive.CustomFields
```

Comment:
This type has been replaced with `LongCustomField`, no need to recreate it.

Reply:
Okay, I will remove this file since it's no longer needed.